litani

```diff
@@ -371,7 +388,7 @@ def positive_int(arg):
     except ValueError as e:
         raise argparse.ArgumentTypeError(
             "Timeout '%s' must be an int" % arg) from e
-    if ret <= 0:
+    if ret < 0:
         raise argparse.ArgumentTypeError(
             "Timeout '%d' must be > 0" % ret) from e
```

Comment:
Should you update the error to say must be `>= 0`?

Reply:
Done